src/babashka/neil.clj

```diff
@@ -381,11 +384,29 @@
    :user
    :version])
 
+(defn- built-in-template? [template]
+  (contains? (set (map name (keys (ns-publics 'org.corfield.new)))) template))
+
+(defn- github-repo-url [lib]
+  (str "https://github.com/" (clean-github-lib lib)))
+
+(defn- add-template-deps [template opts]
+  (let [lib (edn/read-string template)
```

Comment:
Since it's possible to have multiple `template.edn` files in the same `deps.edn` project, it might be worth having a separate override option for the `lib` instead of inferring the `lib` from the `template` all the time.

Reply:
I'm not exactly sure what you mean by this. Could you elaborate or point to a test / doc updates in this PR which explain this?

Reply:
No problem. See [this example from the deps-new README](https://github.com/seancorfield/deps-new#more-general-usage). I split it into multiple lines for readability:
```shell
clojure \
  -Sdeps '{:deps {com.acme.project/cool-lib COORDINATES}}' \
  -Tnew create \
  :template com.acme.project/cool-lib \
  :name myusername/mynewproject
```
In theory you could change the first instance of `com.acme.project/cool-lib` to a different name and the example would still work as long as the directory structure in `:paths` of the lib matches the `:template` option (`resources/com/acme/project/cool_lib`).

This could be useful if you had multiple templates in the same lib which you could then switch between using `:template`. That said, it's possible to achieve the same thing by making a separate lib for each template, so it may not be necessary to support this.

Reply:
Let's not support things we're not certain we're going to need, so let's wait with this.